src/main/java/com/microsoft/azure/batch/TaskOperations.java

```diff
@@ -123,7 +151,8 @@ public Exception getException() {
         /**
          * Submits one chunk of tasks to a job.
          * 
-         * @param taskList A list of {@link TaskAddParameter tasks} to add.
+         * @param taskList
+         *            A list of {@link TaskAddParameter tasks} to add.
          */
         private void submit_chunk(List<TaskAddParameter> taskList) {
```

Comment:
Remove the 'private' modifier for the private variable of private class workingThread.